### ISSUE TITLE
Developer - Account - Connect Account page with API

### ISSUE THREAD

**Issue by Mcscott142** [revelrylabs/bn-mobile-react#243]
## Background 
This ticket is for connecting a users Account to the API. Also, the hiding.showing of the Doorman feature set. 

## Scenario: Signing up for Big Neon and updating Account
Given I have signed up for an Account on Mobile
- [ ] When I update Account Information
- [ ] Then that info is posted via API

## Scenario: A user has account info updated via Web App
Given I have updated info through the Web App
- [ ] When I sign into the Big Neon mobile app
- [ ] Then my account info matches up with the web app

_The doorman scenario, previously written here, was broken out into #309_

**Comment by jwietelmann** [comment]
Brad says that hypothetically, if he remembers correctly, what's left is:

* Updating account info from mobile app
* Displaying account info instead of hard coded values (we are already fetching real account data)

**Comment by jwietelmann** [comment]
Working on the update stuff

**Comment by jwietelmann** [comment]
I think I have it working, but I don't think the password update works on the web side. To be honest I think this might be a kind of bad UI in which to update a password anyway. Usually you'd use a separate form for that.

@zmelamed Should I just remove the password from the account screen for this go-around?

**Comment by jwietelmann** [comment]
actually... @zmelamed I'm calling my shot. I'm removing the password input from the account screen. If the web API starts working, we can try putting it back. Until then it's turned off.

**Comment by jwietelmann** [comment]
Currently stuck on the doorman bit. Not sure how to make myself a doorman in order to test. I don't even know what the "you are a doorman" flag in the data will look like.

**Comment by jwietelmann** [comment]
@Mcscott142 Please unblock somehow. In the meantime I'm doing code reviews.

**Comment by Mcscott142** [comment]
@jwietelmann I'm willing to bet it's something with this line right here from the Web App: https://github.com/big-neon/bn-web/blob/master/src/stores/user.js#L299. 

```
get hasBoxOfficeAccess() {
	return this.isOrgMember || this.isOrgOwner || this.isAdmin; //TODO this permission might come from scopes later
	}
}
```

@zmelamed or @Krakaw Can one of you confirm this will be the check for whether a user has access as a Doorman? 

@jwietelmann feel free to break this out into another issue until we get an answer if this is the only thing holding you up.

**Comment by daybreaker** [comment]
orgMember might be what we need? Though that still might allow too many permissions compared to what they want to allow "doormen" to have.

**Comment by jwietelmann** [comment]
I broke out the doorman bit and put it in the Up Next column, @Mcscott142. Moving this over to QA.

**Comment by Krakaw** [comment]
I need to confirm, but in the user profile of you have, OrgMember, OrgOwner, Admin you'll have access. However if you hit the `server.events.checkins` endpoint and get a 200 status and data result you have the authority to scan people in.

**Comment by zmelamed** [comment]
@Krakaw I think the key here is an endpoint that returns events that a user has authority to manage, and using that to determine what actions can be taken (this is seperate from the normal events index call). If the managed events endpoint returns empty we can just have an empty state for the doorman features that says "This account does not have the authority to manage any events at this time." Is that what `server.events.checkins` is?

**Comment by Krakaw** [comment]
@zmelamed of the `server.events.checkins` returns a 4XX it means they don't have the authority to check-in.

**Comment by zmelamed** [comment]
@Krakaw But how do they show the list of events that they have the authority to check people into?

**Comment by zmelamed** [comment]
@Krakaw nevermind, I just checked the docs, seems that's exactly what we need.
